auto_add_names.py

```diff
@@ -0,0 +1,35 @@
+## This is a small script to automatically add your name in the list alphabetically
+
+import re
+import os
+
+##part 1- Get the info
+print('Welcome Women Githuber')
+name=input('Enter the name you wish to see on the list::')
```

Comment:
Yes, it indeed failed, using raw_input it gives error in python3 but works fine in 2.x, what else can be done about it?